pulsar-config-map.yaml

```diff
@@ -16,8 +15,6 @@ data:
           enabled: true
           consumerConfig: 
             topicNames: "marchtwelve"
-
-
-
-
-
+        admin: 
+          adminConfig: # Accepts the same key-value pair configurations as pulsar client: https://pulsar.apache.org/reference/#/4.0.x/client/client-configuration-client
```

Comment:
TODO - when we work on the doc, let's make sure we have a section to provide guidelines on what to specify for serviceUrl and other configurations that differ between PulsarClient and PulsarAdmin.